contributors/design-proposals/cpu-manager.md

```diff
@@ -158,6 +163,26 @@ node. Once allocated at pod admission time, an exclusive CPU remains
 assigned to a single container for the lifetime of the pod (until it
 becomes terminal.)
 
+##### Configuration
+
+Operators can set the number of CPUs that pods may run on through a new
+Kubelet configuration setting `--cpu-manager-static-num-cpus`, which
```

Comment:
This seem in conflict with CPU Allocatable.  If I set `--cpu-manager-static-num-cpus=2` on a node with CPU Allocatable of 4, the scheduler thinks there are 4 allocatable cores.  However if it tries to schedule more that 2 cores worth of pinned pods, those pods... bounce? It isn't clear what happens.

What is the motivation for exposing this to users?  I thought we talked about implicitly setting this to `Capacity - Kube Resv - System Resv`.